docs/ref/django-admin.txt

```diff
@@ -1863,10 +1863,46 @@ color-coded output if your terminal supports ANSI-colored output. It
 won't use the color codes if you're piping the command's output to
 another program unless the :option:`--force-color` option is used.
 
-Under Windows, the native console doesn't support ANSI escape sequences so by
-default there is no color output. But you can install the `ANSICON`_
-third-party tool, the Django commands will detect its presence and will make
-use of its services to color output just like on Unix-based platforms.
+Windows support
+~~~~~~~~~~~~~~~
+
+On Windows 10, the `Windows Terminal`_ application, `VS Code`_, and `PowerShell where
+virtual terminal processing is enabled`__ support colored output are supported by
+default.
+
+Under Windows, the legacy ``CMD.exe`` native console doesn't support ANSI
+escape sequences so by default there is no color output. In this case either of
+two third-party libraries are needed:
+
+* Install colorama_, a Python package that translates ANSI color codes into
+  Windows API calls. Django commands will detect its presence and will make use
+  of its services to color output just like on Unix-based platforms.
+  ``colorama`` can be installed via pip::
+
+    ...\> py -m pip install colorama
+
+* Install `ANSICON`_, a third-party tool that allows ``CMD.exe`` to process
+  ANSI color codes. Django commands will detect its presence and will make use
+  of its services to color output just like on Unix-based platforms.
+
+Other modern terminal environments on Windows, that support terminal colors,
+but which are not automatically detected as supported by Django, may "fake" the
+installation of ``ANSICON`` by setting the appropriate environmental variable,
+``ANSICON="on"``.
```

Comment:
`:envvar:`

Reply:
@felixxm Where's this linking to? 

```
WARNING: 'envvar' reference target not found: ANSICON="on"
```

Reply:
`:envvar:` won't work unless the docs have `.. envvar:: ANSICON` defined.

We've used it for those that are defined in Django and Python documentation, e.g. fbdb032de266ba5f82e061ab204f6c622889d563.

Reply:
Right, my mistake.